### PR TITLE
[Common Architecture] Implements Account 

### PR DESCRIPTION
**Pull request by sirano11**
## Proposed changes

This PR introduces implementations Account class.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #97

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: core/src/main/java/com/klaytn/caver/account/Account.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+        return new Account(address, accountKey);
+    }
+
+    /**
+     * Set 1 to WeightedMultiSigOptions values(threshold, weights)
+     * @param arrayLength Length of public key array
+     * @return WeightedMultiSigOptions
+     */
+    private static WeightedMultiSigOptions getDefaultOptions(int arrayLength) {
```
Comment (jimni1222) [inline core/src/main/java/com/klaytn/caver/account/Account.java:192]: How about implement this fucntion to WeightedMultiSigOptions class?

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/account/Account.java]: Okay. I added function in WeightedMultiSigOptions class

File: core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java [hunk 2]
```diff
@@ -43,6 +43,47 @@ public WeightedMultiSigOptions(BigInteger threshold, List<BigInteger> weights) {
         this.weights = weights;
     }
 
+
+    /**
+     * Create a WeightedMultiSigOptions instance fill with default value.(threshold, weight has 1)
+     * @param publicKeyArr public key array
+     * @return WeightedMultiSigOptions
+     */
+    public static WeightedMultiSigOptions fillWeightedMultiSigOptionForMultiSig(String[] publicKeyArr) {
```
Comment (jimni1222) [inline core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java:52]: I think function name is different with what this function does.
This function does not fill anything.

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java:52]: How about fillDefaultOptionsValue() for this method name?

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java:52]: I renamed method fillDefaultValuesForWeightedMultSig and fillDefaultValuesForWeightedForRoleBased

Comment (jimni1222) [reply core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java]: I think if you want to name the function with `fillxxx`, you have to get some target as a parameter of this function and fill in the default options here.

However, this is a function that returns default WeightedMultiSigOptions depending on the type of AccountKey, so what if we name it getDefaultOptionsForWeightedMultiSig? 

If i misunderstand this function, then please let me know.

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java]: Okay, I modified this function name.

**Pull request merged**